### ISSUE TITLE
docker compose on M1 Mac

### ISSUE THREAD

**Issue by tomribaroff** [iusztinpaul/energy-forecasting#24]
When running the line:

docker compose --env-file .env up --build -d

I am getting the error:

no matching manifest for linux/arm64/v8 in the manifest list entries

I believe this problem is because the code is written to be run on an Ubuntu system, which defaults to a different image type, than on my Mac. 

https://forums.docker.com/t/run-x86-intel-and-arm-based-images-on-apple-silicon-m1-macs/117123

When I try to add a line to the docker-compose.yaml file to specific a different platform, I get an error. Any ideas how to fix this?

**Comment by tomribaroff** [comment]
<img width="520" alt="Screenshot 2024-01-10 at 4 36 01 PM" src="https://github.com/iusztinpaul/energy-forecasting/assets/54896315/00e05ae3-515b-4d1f-9546-5a126c962116">

**Comment by iusztinpaul** [comment]
Hello @tomribaroff,

Your intuition makes sense. I never got the following error as I developed the course on an Ubuntu system, but I know people who went through the course on M chips and did not have this issue. 

So, I don't think that is the issue.

But after searching `no matching manifest for linux/arm64/v8 in the manifest list entries` on google I found this link: https://stackoverflow.com/questions/65456814/docker-apple-silicon-m1-preview-mysql-no-matching-manifest-for-linux-arm64-v8

Let me know if it helps.
